charts/velero/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.3.1
 description: A Helm chart for velero
 name: velero
-version: 2.9.15
+version: 2.9.16
```

i would say to bump to 
```suggestion
version: 2.10.0
```
since this is not a bug fix

---

done!